src/views/registration/SubSchema.js

```diff
@@ -35,17 +34,19 @@ define(['okta', 'util/PasswordComplexityUtil'], function (Okta, PasswordComplexi
   });
 
   return Okta.View.extend({
-    className: 'password-complexity',
+    className: 'subschema',
 
     children: function () {
-      return _.map(this.passwordComplexity.enabledComplexities, function(complexityName) {
-        var complexityValue = this.get(complexityName);
-        var message = PasswordComplexityUtil.complexities[complexityName].getI18nMessage(complexityValue);
-        return PasswordComplexity.extend({
-          name: complexityName, 
+      return this.subSchemas.map(function(subSchema, index) {
+        var description = subSchema.get('description');
+        // TODO API should send translated strings instead of i18n code inside description
```

Comment:
There is a JIRA to track the API changes that will send the description as a translated string. 
https://oktainc.atlassian.net/browse/OKTA-142209